README.md

```diff
@@ -17,65 +17,23 @@ multi-dimensional labeled arrays, such as:
 -   Calculating statistics (e.g., "climatology") across distributed datasets
     with arbitrary groups.
 
-Xarray-Beam is implemented as a _thin layer_ on top of existing libraries for
-working with large-scale Xarray datasets. For example, it leverages
-[Dask](https://dask.org/) for describing lazy arrays and for executing
-multi-threaded computation on a single machine.
+For more about our approach and how to get started,
+**[read the documentation](https://xarray-beam.readthedocs.io/)**!
 
 **🚨 Warning: Xarray-Beam is new and unpolished 🚨**
 
 Expect sharp edges 🔪 and performance cliffs 🧗, particularly related to the
 management of lazy data with Dask and reading/writing data with Zarr. We have
-used it to efficiently process 5 TB datasets. We _expect_ it to scale to PB size
-datasets but that's easier said than done. We welcome feedback and contributions
-from early adopters, and hope to have it ready for wider audience soon.
+used it to efficiently process ~25 TB datasets. We _expect_ it to scale to PB
+size datasets but that's easier said than done. We welcome feedback and
```

Comment:
style nit (and my grammar may be wrong here): add a comma between "datasets" and "but".